source/includes/administration/_commitments.md

```diff
@@ -467,4 +467,71 @@ Attributes | &nbsp;
     "taskId": "75a68ec2-d63e-4bc9-b485-8d5a3cc33d4f",
     "taskStatus": "SUCCESS"
 }
-```
\ No newline at end of file
+```
+
+<!-------------------- UPDATE COMMITMENT -------------------->
+### Update commitment
+
+`PUT /commitments/:id`
+
+Update the attributes of a commitment for an organization
+
+A commitment that is applied can be updated. Meaning, that since an invoice hasn't been issued to the customer, all of the attributes of the commitment can be updated.
+
+A commitment that is pre-paid can be updated, **but** it will be terminated and a new commitment will be created with the updated attributes and will be effective at the start at the end of the current billing cycle.
+
+Updating a commitment always requires a reprocess from the start of the current billing cycle, expect if only the name has been updated.
+
+```shell
+# update commitment
+curl "https://cloudmc_endpoint/api/v2/commitments/:id" \
+   -X PUT -H "MC-Api-Key: your_api_key" -d '{ ... }'
+```
+
+> Request body example
+
+```json
+{
+   "id": "60d16762-bec2-4d5a-916d-15c81b6c2d98"
+   "name": "updated fixed monthly price commitment",
+   "committedProducts": [
+      {
+         "product": {
+            "id": "73ebf9c8-ecf7-4ee6-b881-150023d5f422"
+         },
+         "committedAmount": 1234.0
+      }
+   ],
+   "pricingMethod": "FIXED_PRICE",
+   "type": "RESOURCE",
+   "fixedPrice": 550.0,
+   "startDate": "2022-06-01T00:00:00Z",
+   "endDate": "2023-01-01T00:00:00Z"
+}
+```
+
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+    "taskId": "75a68ec2-d63e-4bc9-b485-8d5a3cc33d4f",
+    "taskStatus": "SUCCESS"
+}
+```
+
+Attributes | &nbsp;
+---- | ---
+`id`<br/>*string* | The identifier of the commitment.
+`name`<br/>*string* | The name of the commitment.
+`organization`<br/>*Organization* | The organization that is associated to the commitment.
+`reseller`<br/>*Organization* | The organization that owns the commitment.
+`committedProducts`<br/>*Array[CommittedProduct]* | The list of products, commitment levels and negotiated price associated to the commitment.
+`pricingMethod`<br/>*enum* | The pricing method of the commitment. Possible values are: `FIXED_PRICE` and `UTILITY_DISCOUNT`. Used to define how the effective price of the commitment is calculated.
```

Comment:
We'll need to add the tax code here as well once @lniemann-co merges his PR

Reply:
Pre-emptively done 130920581335d8572ed8f1c5ff2ebb5dd6004ad0